### PR TITLE
Refactors freedom implants to use emote signals

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[CLEANLINESS] [INTERNAL] [GAME OBJECTS]
## About the PR 
Also makes them try not to conflict with each other.
Doesn't touch the derringer code because I couldn't think of a good way to deal with equipping/unequipping clothing (atom property maybe?) also doesn't touch patho because patho is not being maintained.
Adds `waitfor = FALSE` to both `mob/emote` and `mob/living/carbon/human/emote` to pacify the linter, I don't _think_ that will break anything?

## Why's this needed? 
Cleaner code, makes it easier to add more emote triggered things in future.

### CODE REVIEW

File: code/mob/living/carbon/human/procs/emote.dm [hunk 1]
```diff
@@ -2277,6 +2297,7 @@
 // that would be difficult, datumize emotes 2day!
 #ifdef SPACEMAN_DMM
 /mob/living/carbon/human/emote(var/act, var/voluntary = 0, var/emoteTarget = null)
+	..()
```
Comment (BatElite) [inline code/mob/living/carbon/human/procs/emote.dm:2300]: I think this line might be what's screwing up the linter

Comment (TobleroneSwordfish) [reply code/mob/living/carbon/human/procs/emote.dm]: oh no it's worse than that

**Comment by TobleroneSwordfish** [comment]
asjdajksl I'll figure out how to defeat the linter tomorrow

File: code/mob.dm [hunk 2]
```diff
@@ -231,6 +231,8 @@
 	var/radiation_dose_decay = 0.02 //at this rate, assuming no lag, it will take 40 life ticks, or ~80 seconds to recover naturally from 1st stage radiation posioning,
 	/// set to observed mob if you're currently observing a mob, otherwise null
 	var/mob/observing = null
+	/// A list of emotes that trigger a special action for this mob
+	var/list/trigger_emotes = null
```
Comment (frawhst) [inline code/mob.dm:235]: This feels gross but I don't have a better idea so 👍

**Pull request merged**